src/actions/summit-actions.js

```diff
@@ -76,6 +76,7 @@ export const getSummitById = (summitId) => async (dispatch) => {
       "badge_features_types," +
       "badge_access_level_types," +
       "badge_view_types," +
+      "order_extra_questions," +
```

Comment:
please avoid this expand ... it has a great cost
and u could remove this 
https://github.com/fntechgit/summit-admin/blob/547a7f1eb9824bd9d793a88ddff9dc67eb1a4b58/src/pages/orders/edit-order-extra-question-page.js#L121
and do the proper expand here https://github.com/fntechgit/summit-admin/blob/1181e370e64eb8c3665014492df0b1cf68345207/src/components/forms/extra-question-form/index.js#L161 ( sub_question expand on getting the entity)